app/views/teaching_events/show/_train-to-teach-event-summary.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="about-ttt-event">
+  <p>Watch presentations, ask questions, speak to advisers one-to-one.</p>
+</div>
+
+<figure>
+  <blockquote>
+    Absolutely loved this day. Very informative, inspiring and I'm
```

Comment:
Please can you change this quote to:

It was exactly what I needed to give me guidance on the next steps to be taken